nextflow.config

```diff
@@ -62,7 +62,7 @@ params {
 
     // GAS Call
     gm_thresholds = "10,5,0"
-    gm_delimiter = "'.'" // note the single quotes surrounding the delimiter
+    gm_delimiter = "." // note the single quotes surrounding the delimiter
```

Comment:
Does the comment need to be updated now that the single quotes are gone?

Reply:
Ah yes! Removed here: [b449215](https://github.com/phac-nml/gasnomenclature/pull/11/commits/b449215fab5254e1ac3d26307efe63694dbd71ef)